test/test_jit.py

```diff
@@ -1441,6 +1441,57 @@ def constant_prop():
         self.run_pass('constant_propagation', constant_prop.graph)
         self.assertExpected(canonical(constant_prop.graph))
 
+    def test_export_dropout(self):
+        class DropoutTest(torch.nn.Module):
+            def __init__(self):
+                super(DropoutTest, self).__init__()
+                self.dropout = torch.nn.Dropout()
+
+            def forward(self, x):
+                return self.dropout(x)
+
+        test = DropoutTest()
+        test.eval()
+
+        traced = torch.jit.trace(test, (torch.rand(3, 4),), check_trace=False)
+        imported = self.getExportImportCopy(traced)
+        x = torch.randn(3, 4)
+        self.assertEqual(traced(x), imported(x))
+
+    def test_export_batchnorm(self):
+        for mode in ['eval', 'train']:
+            class BatchnormTest(torch.nn.Module):
+                def __init__(self):
+                    super(BatchnormTest, self).__init__()
+                    self.batchnorm = torch.nn.BatchNorm1d(100)
```

Comment:
also test `torch.nn.BatchNorm1d(100, affine=False)`

Also test `torch.nn.BatchNorm2d(100)` and `torch.nn.BatchNorm2d(100, affine=False)`